xclim/ensembles/_robustness.py

```diff
@@ -49,7 +50,7 @@ def change_significance(  # noqa: C901
         a distribution across the future period.
         `fut` and `ref` must be of the same type (Dataset or DataArray). If they are
         Dataset, they must have the same variables (name and coords).
-    test : {'ttest', 'welch-ttest', 'mannwhitney-utest', 'brownforsythe-test', 'threshold', None}
+    test : {'ttest', 'welch-ttest', 'mannwhitney-utest', 'brownforsythe-test', 'ipcc-advanced', 'threshold', None}
```

Comment:
I'd be more specific regarding naming, qqch comme `ipcc-ar6-c`.

Reply:
Also, let's say we wanted to implement later the first gamma calculation with preindustrial data, would we have to break anything? Should we call it `ipcc-ar6-c-hist` and later implement `ipcc-ar6-c-pi` ? or something ?

Reply:
Agreed to change `ipcc-advanced` to `ipcc-ar6-c`. 

Yes it would be breaking as this function would now need 3 inputs instead 2. : future, reference (for the deltas) and preindustrial (for `gamma`).

In fact, we could implement both as another function : `gamma(ref, input_type='pi'/'hist')` (or maybe a better name). The `ipcc-ar6-c` test in the main function would simply call `gamma(ref, 'hist')` while a user with PI data would call:
```python3
change_significance(fut, ref, test='threshold', abs_thresh=gamma(pi_data, 'pi'))
```
Does that make sense ?

Reply:
I _think_ having a third input (preindustrial) would be more user friendly. I'd implement the `gamma` function as a private function.